### PR TITLE
Bunch'o'fixes

### PR DESCRIPTION
**Pull request by MikhailArkhipov**

### CODE REVIEW

File: src/Package/Impl/Help/ShowHelpOnCurrentCommand.cs [hunk 1]
```diff
@@ -88,7 +88,7 @@ internal sealed class ShowHelpOnCurrentCommand : PackageCommand {
             if (activeReplWindow != null && _activeReplTracker.IsActive) {
                 return activeReplWindow.InteractiveWindow.TextView;
             }
-            return _textViewTracker.GetLastActiveTextView(RContentTypeDefinition.ContentType);
+            return _textViewTracker.LastActiveTextView;
```
Comment (AlexanderSher) [inline src/Package/Impl/Help/ShowHelpOnCurrentCommand.cs]: What if it was SQL script?

Comment (MikhailArkhipov) [inline src/Package/Impl/Help/ShowHelpOnCurrentCommand.cs]: It is context menu command so the context menu itself wouldn't show up.

Comment (MikhailArkhipov) [inline src/Package/Impl/Help/ShowHelpOnCurrentCommand.cs]: I.e. if context menu with this command is up, it is the right content type

**Comment by AlexanderSher** [comment]
:+1:

**Pull request merged**